examples/rvc-app/rvc-common/rvc-app.matter

```diff
@@ -1115,6 +1115,296 @@ cluster RvcOperationalState = 97 {
   command GoHome(): OperationalCommandResponse = 128;
 }
 
+/** The Service Area cluster provides an interface for controlling the locations where a device should operate, and for querying the current location. */
+cluster ServiceArea = 336 {
+  revision 1; // NOTE: Default/not specifically set
```

Comment:
please set the revision explicitly in the XML.

something like `<globalAttribute side="either" code="0xFFFD" value="1"/>`

Reply:
This change is likely to cause some changes in auto-generated files and increase the size of this PR. To try to limit the size of this PR and given that there are a number of changes that will need to be made to the XML following spec changes, I have created a [separate issue](https://github.com/project-chip/connectedhomeip/issues/34580) to track this change. We'll implement this as we're updating the XML to the current spec changes.